auth/utils.py

```diff
@@ -0,0 +1,15 @@
+import jwt
+
+SECRET_KEY = '1234567890'  # TODO: Move to environment variables
+
+def is_valid_password(password):
+    return len(password) > 5
+
+def authenticate(u, p):
+    if u == "demo@example.com" and p == "password123":
+        return {"email": u}
+    return None
```

Comment:
change email parameter name u to e

```suggestion
def authenticate(e, p):
    if e == "demo@example.com" and p == "password123":
        return {"email": e}
    return None
```